conduit/pipeline/pipeline.go

```diff
@@ -264,14 +264,21 @@ func (p *pipelineImpl) Init() error {
 	importerName := (*p.importer).Metadata().Name
 	importerLogger.SetFormatter(makePluginLogFormatter(plugins.Importer, importerName))
 
+	// InitProvider
+	round := sdk.Round(p.pipelineMetadata.NextRound)
+	// Initial genesis object is nil--gets updated after importer.Init
+	var initProvider data.InitProvider = conduit.MakePipelineInitProvider(&round, nil)
+	p.initProvider = &initProvider
+
 	configs, err := yaml.Marshal(p.cfg.Importer.Config)
 	if err != nil {
 		return fmt.Errorf("Pipeline.Start(): could not serialize Importer.Args: %w", err)
 	}
-	genesis, err := (*p.importer).Init(p.ctx, p.makeConfig("importer", importerName, configs), importerLogger)
+	genesis, err := (*p.importer).Init(p.ctx, *p.initProvider, p.makeConfig("importer", importerName, configs), importerLogger)
```

Comment:
I wanted to use the initProvider since it's also passed into the processors and exporters during Init. This would give us a uniform interface, although you're right all we need is the round so it would be simpler to pass that in instead.